packages/syft/src/syft/core/tensor/autodp/row_entity_phi.py

```diff
@@ -248,35 +250,40 @@ def __and__(self, other: Any) -> RowEntityPhiTensor:
             rows=[x & other for x in self.child], check_shape=False
         )
 
-    def __truediv__(  # type: ignore
+    def __truediv__(
         self, other: Union[RowEntityPhiTensor, AcceptableSimpleType]
     ) -> RowEntityPhiTensor:
 
-        if is_acceptable_simple_type(other) or len(self.child) == len(other.child):  # type: ignore
+        if is_acceptable_simple_type(other) or len(self.child) == len(other.child):
             new_list = list()
             for i in range(len(self.child)):
                 if is_acceptable_simple_type(other):
                     new_list.append(self.child[i] / other)
                 else:
-                    new_list.append(self.child[i] / other.child[i])  # type: ignore
+                    new_list.append(self.child[i] / other.child[i])
             return RowEntityPhiTensor(rows=new_list, check_shape=False)
         else:
             raise Exception(
-                f"Tensor dims do not match for __truediv__: {len(self.child)} != {len(other.child)}"  # type: ignore
+                f"Tensor dims do not match for __truediv__: {len(self.child)} != {len(other.child)}"
             )
 
     def repeat(
-        self, repeats: Union[int, Tuple[int, ...]], axis: Optional[int] = None
+        self, repeats: Union[int, List[int]], axis: Optional[int] = None
     ) -> RowEntityPhiTensor:
 
+        if not isinstance(repeats, int):
+            raise Exception(
+                f"{type(self)}.repeat for repeats: List {repeats} not implemented yet"
+            )
+
         if axis is None:
             raise Exception(
                 "Conservatively, RowEntityPhiTensor doesn't yet support repeat(axis=None)"
             )
 
         if axis == 0 or axis == -len(self.shape):
             new_list = list()
-            for r in range(repeats):  # type: ignore
+            for _ in range(repeats):
```

Comment:
This was causing an error because one valid input is a Tuple for repeats, so we need to guard against that until its implemented with an Exception as above.